crates/k8s-version/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Replace `lazy_static` with `std::cell::LazyCell` ([#827]).
```

Comment:
nit: I think we should also include the link to _this_ PR in all changelogs.

Reply:
98282dd

Reply:
`crates/k8s-version/CHANGELOG.md` still needs to mention _this_ PR.

Reply:
Good catch! 1e2cff3